From: Jeff Mahoney <jeffm@suse.com>
Subject: [PATCH] ocfs2/dlm: fix compilation on ia64
Patch-mainline: Submitted 2006/01/09

 Including <asm/signal.h> results in compilation failure on ia64 due to
 a missing <linux/compiler.h>

 Including <linux/signal.h> corrects the problem.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>

diff -ruNpX dontdiff linux-2.6.15/fs/ocfs2/dlm/userdlm.c linux-2.6.15-fix/fs/ocfs2/dlm/userdlm.c
--- linux-2.6.15/fs/ocfs2/dlm/userdlm.c	2006-01-08 23:19:50.000000000 -0500
+++ linux-2.6.15-fix/fs/ocfs2/dlm/userdlm.c	2006-01-09 12:31:08.056894624 -0500
@@ -27,7 +27,7 @@
  * Boston, MA 021110-1307, USA.
  */
 
-#include <asm/signal.h>
+#include <linux/signal.h>
 
 #include <linux/module.h>
 #include <linux/fs.h>
